tests/test_linked_pa_templates.py

```diff
@@ -80,15 +80,16 @@ def test_c_update_linked_pa_template(self):
 
         self.assertEqual(response[1], 200, "Response should be 200 - Success")
         self.assertEqual(type(response[0].data), dict, "Response should be of Dictionary type.")
-        self.assertEqual(type(response[0].data[template_id]),
-            LinkedPATemplateSummary, "Response should be of LinkedPATemplateSummary type.")
+        self.assertEqual(type(response[0].data),
+            dict, "Response should be of LinkedPATemplateSummary type.")
```

Comment:
looks like you are checking for dict type twice and the validation message is different

Reply:
will change the validation message to dict type, missed this.